listenbrainz/webserver/static/js/src/LastFMImporter.tsx

```diff
@@ -162,7 +161,9 @@ export default class LastFmImporter extends React.Component<
       console.warn(
         `${reason} while fetching last.fm page=${page}, retrying in 3s`
```

Comment:
would be nice to make this comment and the 3000ms timer in the next line read from the same variable so that we can adjust it as needed